phpBB/styles/prosilver/theme/common.css

```diff
@@ -3,8 +3,6 @@
 /* -------------------------------------------------------------- */
 /* stylelint-disable selector-max-compound-selectors */
 /* stylelint-disable selector-no-qualifying-type */
-/* stylelint-disable declaration-property-unit-whitelist */
-/* stylelint-disable declaration-property-unit-blacklist */
```

Comment:
Would you think that the issue would be solved if I revert the files? Okay, Then I'll give a try, Thanks a lot for your support.